### ISSUE TITLE
Add an option to disable history sync

### ISSUE THREAD

**Issue by rscipher001** [a85/POSTMan-Chrome-Extension#10398]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

Technically not related to a problem

### Describe the solution you'd like

It would be good if I can disable history syncing or history option completely, I don't want it and I don't need it but there is no way to pause it or completely disable it.

**Comment by giridharvc7** [comment]
@SecureSnowball Can you help us understand why you need this feature?

**Comment by rscipher001** [comment]
Yes, Postman fetches history from servers and no offense but your servers are slow that makes history feature kinda useless, it seems like it exists only for you but no use for us.

**Comment by giridharvc7** [comment]
Thanks for the candid feedback :) 
In the current setup, history loads only when you explicitly open the history tab and is not Pre-fetched. 

> it seems like it exists only for you but no use for us.

Sorry about this. Can you help me understand how you intend to use history? This will help us build a better performant system that puts you first!
